### PR TITLE
markdown lint, typo

### PR DESCRIPTION
**Pull request by maddhruv**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
+# Contributing
+
 ## Basics of Contributing
 
 - Use descriptive commit messages
 - Run tests with `npm test` before submitting a PR / pushing
   - Resolve issues if tests surface them
 
-## How Can You Contribute?
+## How Can You Contribute
```
[the remark this answers is not in the document]
Comment (maddhruv) [reply CONTRIBUTING.md]: the linter says
`no trailing punctuations in heading`

[the remark this answers is not in the document]
Comment (bnb) [reply CONTRIBUTING.md]: boo.

Well in that case we'll need to rephrase:

```
How You Can Contribute
```

[the remark this answers is not in the document]
Comment (maddhruv) [reply CONTRIBUTING.md]: done

**Comment by bnb** [comment]
Going to merge this as a part of 0.15.0 👍

**Pull request merged**